python/paddle/fluid/layers/control_flow.py

```diff
@@ -1165,20 +1165,27 @@ def equal(x, y, cond=None):
     This layer returns the truth value of :math:`x == y` elementwise.
 
     Args:
-        x(Variable): First operand of *equal*
-        y(Variable): Second operand of *equal*
-        cond(Variable|None): Optional output variable to store the result of *equal*
+        x(Variable): Tensor, data type is float32, float64, int32, int64.
+        y(Variable): Tensor, data type is float32, float64, int32, int64.
+        cond(Variable, optional): Optional output which can be any created 
+            Variable that meets the requirements to store the result of *equal*.
+            if cond is None, a new Varibale will be create to store the result.
```

Comment:
a new Varibale will be created to store the result

Reply:
Done